### PR TITLE
147 clean notifications

### PR DESCRIPTION
**Pull request by Ycaro02**

### CODE REVIEW

File: src/angularjs/src/app/services/websocket/notification/service.ts [hunk 1]
```diff
@@ -123,13 +152,20 @@ export class NotificationService {
...
+		}
 	}
 
 	updateNewNotification(notification: any)
 	{
 		const notif = this.getNotification(notification);
 		this.notif.push(notif);
+		if (notification.status === NotifStatus.NOTSEEN)
+			this.notif_not_seen++;
 		console.log(notif);
```
Comment (Pixailz) [inline src/angularjs/src/app/services/websocket/notification/service.ts:169]: lil console.log() to remove :heart:

**Review by Pixailz — CHANGES_REQUESTED** [review]
just a little console.log to remove

Good to you two @RuBeeN-cmd and @Ycaro02 :)

**Review by RuBeeN-cmd — APPROVED** [review]

File: src/angularjs/src/app/services/websocket/notification/service.ts (hunk 1 above)
Comment (RuBeeN-cmd) [reply src/angularjs/src/app/services/websocket/notification/service.ts:169]: Done

**Review by Ycaro02 — COMMENTED** [review]
Nice !

**Review by Pixailz — APPROVED** [review]

**Pull request merged**